plasma_framework/contracts/src/exits/payment/controllers/PaymentChallengeIFEOutputSpent.sol

```diff
@@ -69,7 +69,7 @@ library PaymentChallengeIFEOutputSpent {
 
         verifyInFlightTransactionStandardFinalized(controller, args);
         verifyOutputType(controller, args);
-        verifyChallengingTransactionSpendsOutput(controller, args, ife);
+        verifyChallengingTransactionSpendsOutput(controller, args);
```

Comment:
Are you sure that this should me a part of this PR?

I believe I approved those changes earlier today.

Reply:
Solved by a merge.